v1/anbox-cloud-appliance.yaml

```diff
@@ -0,0 +1,21 @@
+description: Anbox Cloud Appliance
+version: latest
+
+instances:
+  anbox-cloud-appliance:
+    limits:
+      min-cpu: 4
+      min-mem: 4G
+      min-disk: 50G
+    timeout: 900
+    cloud-init:
+      vendor-data: |
+        runcmd:
+        - apt update
+        - DEBIAN_FRONTEND=noninteractive apt purge -y linux-kvm linux-image-kvm linux-headers-kvm linux-image-* linux-headers-* linux-modules-*
+        - apt install -y linux-image-generic linux-headers-generic
+        - apt autoremove -y
```

Comment:
Is that necessary? We actually use the generic kernel so it supports the other hypervisors?

Reply:
Jepp, launch a VM on the LXD driver gives the KVM kernel:

```
$ multipass launch
Launched: infallible-wapiti                                                     
$ multipass shell infallible-wapiti
ubuntu@infallible-wapiti:~$ uname -a
Linux infallible-wapiti 5.4.0-1044-kvm #46-Ubuntu SMP Wed Jul 14 21:36:50 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

The KVM kernel sadly doesn't include the binder and ashmem driver we depend on. We slowly get them into all kernels but it takes time...

Reply:
> Jepp, launch a VM on the LXD driver gives the KVM kernel:

Ah indeed, LXD does differently.